apps/myjobs/config/spring.rb

```diff
@@ -0,0 +1,6 @@
+%w(
```

Comment:
We intentionally disabled spring when using Rails 4 because of issues with our deployment environment. I'm not sure we are ready to re-enable spring until we learn more. So we should probably delete this file so it is not executed.

Reply:
done.